include/simdjson/dom/parser.h

```diff
@@ -120,7 +125,7 @@ class parser {
    * Parse a JSON document and return a temporary reference to it.
    *
    *   dom::parser parser;
-   *   element doc = parser.parse(buf, len);
+   *   element doc_root = parser.parse(buf, len);
```

Comment:
The convention I've been following everywhere else in the code is to call that `doc`, IIRC. Now that we're exposing dom::document more specifically, it makes sense to call this something else, but we should probably change it universally when we get time (`doc_root` sounds fine, `root` might be OK too). This change is fine to make right now, though.

Reply:
Noted. It is a good point.